agenta-backend/agenta_backend/models/converters.py

```diff
@@ -113,8 +118,12 @@ async def human_evaluation_db_to_pydantic(
         evaluation_type=evaluation_db.evaluation_type,
         variant_ids=[str(variant) for variant in evaluation_db.variants],
         variant_names=variant_names,
-        testset_id=str(evaluation_db.testset.id),
-        testset_name=evaluation_db.testset.name,
+        testset_id=""
+        if type(evaluation_db.testset) is Link
+        else str(evaluation_db.testset.id),
+        testset_name=""
+        if type(evaluation_db.testset) is Link
+        else str(evaluation_db.testset.name),
```

Comment:
Same concern as [here](https://github.com/Agenta-AI/agenta/pull/1189/files#r1450445400).

Reply:
Due to the same reason as mentioned above